src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanedStyles = sourceString
```

Comment:
Consider adding a check to ensure `sourceString` is a string before proceeding with the operations. This will prevent potential runtime errors if a non-string value is passed to the function.